draft-ietf-rats-corim.md

```diff
@@ -1388,137 +1389,106 @@ phase can be amortised across multiple appraisals.
-(Such policies model the trust relationships between the Verifier Owner and the relevant suppliers, and are out of scope of the present document.)
+The Verifier processes all CoBOMs that are valid at the point in time of Evidence Appraisal and activates all tags referenced therein.
 
+A Verifier MAY decide to discard some of the available and valid CoBOMs depending on any locally configured authorization policies.
+(Such policies model the trust relationships between the Verifier Owner and the relevant suppliers, and are out of the scope of the present document.)
 For example, a composite device ({{Section 3.3 of -rats-arch}}) is likely to be fully described by multiple CoRIMs, each signed by a different supplier.
-In such case, the Verifier Owner may instruct the Verifier to discard tags activated by supplier CoBOMs that are not activated by the trusted integrator.
+In such case, the Verifier Owner may instruct the Verifier to discard tags activated by supplier CoBOMs that are not also activated by the trusted integrator.
 
-After the Verifier has processed all CoBOMs it MUST discard any tags which have
-not been activated by a CoBOM.
+After the Verifier has processed all CoBOMs it MUST discard any tags which have not been activated by a CoBOM.
 
 ### Tags Identification and Validation
 
-The Verifier chooses tags -- including Concise Module ID Tags (CoMID, {{sec-comid}}),
-Concise Software ID Tags (CoSWID, {{-coswid}}),
-and/or Concise Trust Anchor Stores (CoTS, {{-ta-store}}) --
-from the selected CoRIMs.
+The Verifier chooses tags - including Concise Module ID Tags (CoMID, {{sec-comid}}), Concise Software ID Tags (CoSWID, {{-coswid}}), and/or Concise Trust Anchor Stores (CoTS, {{-ta-store}}) - from the selected CoRIMs.
 
-The Verifier MUST discard all tags which are not syntactically and semantically
-valid.
-In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples)
-MUST be successfully resolved.
+The Verifier MUST discard all tags which are not syntactically and semantically valid.
+In particular, any cross-referenced triples (e.g., CoMID-CoSWID linking triples) MUST be successfully resolved.
 
 ### Appraisal Context Construction
 
 All of the validated and potentially useful tags are loaded into the Appraisal Context.
+This concludes the Appraisal Context initialisation phase.
 
 [^issue]: Content missing. Tracked at https://github.com/ietf-rats-wg/draft-ietf-rats-corim/issues/96
 
-This concludes the initialisation phase.
-
-## Evidence Collection
+## Evidence Collection {#sec-ev-coll}
 
-In the Evidence collection phase the Verifier communicates with attesters to
-collect Evidence.
+During the Evidence collection phase, the Verifier communicates with attesters to gather Evidence.
+The first part of this phase does not involve any cryptographic validation.
+This means that Verifiers can use untrusted code for their initial Evidence collection.
```

Comment:
```suggestion
This means that Verifiers MAY use untrusted code for their initial Evidence collection.
```

Reply:
RFC2119 terms are for interop requirements.  This one is an implementation choice instead.